doc/source/whatsnew/v1.1.0.rst

```diff
@@ -582,6 +582,8 @@ I/O
 - Bug in :func:`pandas.io.json.json_normalize` where location specified by `record_path` doesn't point to an array. (:issue:`26284`)
 - :func:`pandas.read_hdf` has a more explicit error message when loading an
   unsupported HDF file (:issue:`9539`)
+- Bug in :meth:`~DataFrame.to_parquet` was not raising ``PermissionError`` when writing to a private s3 bucket with invalid creds. (:issue:`27679`)
+- Bug in :meth:`~DataFrame.to_csv` was silently failing when writing to an invalid s3 bucket. (:issue:`32486`)
```

Comment:
there is a 3rd error that you are closing can you add (either a 3rd note or add to one of the others as appropriate)

Reply:
I actually can't repo the 3rd problem on master. Removed for now & will follow up on issue https://github.com/pandas-dev/pandas/issues/32470#issuecomment-616898365